openhands/agent_server/docker/Dockerfile

```diff
@@ -56,13 +62,76 @@ COPY --from=ghcr.io/astral-sh/uv /uv /uvx /bin/
 RUN set -eux; \
     # Install base packages (works for both Debian-based images)
     apt-get update; \
-    apt-get install -y --no-install-recommends sudo ca-certificates curl wget; \
+    apt-get install -y --no-install-recommends \
+        ca-certificates curl wget sudo apt-utils git jq tmux build-essential \
+        coreutils util-linux procps findutils grep sed \
+        # Docker dependencies
+        apt-transport-https gnupg lsb-release; \
     \
     # Create user and group
     (getent group ${GID} || groupadd -g ${GID} ${USERNAME}); \
-    (id -u ${USERNAME} >/dev/null 2>&1 || useradd -m -u ${UID} -g ${GID} -s /usr/sbin/nologin ${USERNAME}); \
+    (id -u ${USERNAME} >/dev/null 2>&1 || useradd -m -u ${UID} -g ${GID} -s /bin/bash ${USERNAME}); \
+    # Add user to sudo group
```

Comment:
Not sure if we need this, but let's start simple first and add them back when we run into issue.

especially now we don't have such complex dependency now

```suggestion
```